support/docker/build.sh

```diff
@@ -29,11 +29,136 @@ stderr.printf() {
 	printf "$@" >&2
 }
 
-_nix_docker() {
+
+_nix_container_manager() {
+
+	# Ask if the user wants to proceed as root
+	if (($(id -u) == 0)); 
+	then
+		stderr.printf "\nYou don't need to run this script as root.\n\nLear more on how run docker/podman in rootless mode:\n Docker: https://docs.docker.com/engine/security/rootless/\n Podman: https://github.com/containers/podman/blob/main/docs/tutorials/rootless_tutorial.md\n\n"
```

Comment:
```suggestion
		stderr.printf ""
		stderr.printf "You don't need to run this script as root."
		stderr.printf ""
		stderr.printf "Learn more on how run docker/podman in rootless mode:"
		stderr.printf " Docker: https://docs.docker.com/engine/security/rootless/"
		stderr.printf " Podman: https://github.com/containers/podman/blob/main/docs/tutorials/rootless_tutorial.md"
		stderr.printf ""
```

Long spans of text without newlines are harder to read.

Note how `Lear` was changed to `Learn`; easier to spot when reading :)

This is not high performance code, putting multiple printfs one after the other is more correct since it helps the developer more.

Reply:
Thank you very much for the suggestion! 
It really makes a huge difference to make the text easy to read.

This is something I should use pretty much everywhere from now on :)